test/csuite/random_abort/main.c

```diff
@@ -208,19 +220,22 @@ thread_run(void *arg)
          * Decide what kind of operation can be performed on the already inserted data.
          */
         if (i % MAX_NUM_OPS == OP_TYPE_DELETE) {
+            /* FIXME: Temporarily turn off tests for lower isolation levels. */
+            testutil_check(session->begin_transaction(session, "isolation=snapshot"));
+
             if (columnar_table)
                 cursor->set_key(cursor, i);
             else
                 cursor->set_key(cursor, kname);
 
             testutil_check(cursor->remove(cursor));
 
+            testutil_check(session->commit_transaction(session, NULL));
+
             /* Save the key separately for checking later.*/
             if (fprintf(fp[DELETE_RECORD_FILE_ID], "%" PRIu64 "\n", i) == -1)
                 testutil_die(errno, "fprintf");
-        } else if (i % MAX_NUM_OPS == OP_TYPE_INSERT)
```

Comment:
I don't understand the reason behind this change. Can you add a comment or FIXME so it can be fixed in a later ticket that enables it?

Reply:
This change is a cleanup. I just don't like the continue. You want me to revert that?

Reply:
I am fine with it.

Reply:
I am good with this cleanup too.